### ISSUE TITLE
Issues / comments related to Clear Words - Part 4

### ISSUE THREAD

**Issue by johnfoliot** [w3c/silver#582]
**Moving to the "Develop" tab, it states:**

Technical responsibilities

Ensure that inline definitions are accessible by:
- Make inline definitions keyboard accessible
- Once activated (either by keyboard or hover), ensure definitions remain open until the user closes them. Because of hover limitations on mobile, hover states should only act as confirmation that an element is actionable. See BBC Mobile Accessibility Guidelines: Actionable Elements
- Make definitions visually distinct from a link.
- Design one click to the definition and one click return.
- When inline definition are displayed in a modal, all the accessibility requirements of a modal should be included, such as: disable the background text (use a modal display) and ensure that background text is covered or blurred so people aren't confused between foreground and background text, and that the keyboard focus remains in the modal until it is closed.

**Questions:**
- How do these Technical Responsibilities correlate to the stated goal of ensuring: 
  - (Use) Simple tense
  - (No) Literal language
  - Avoid double negatives
  - (No) Nested clauses
  - (Proper use of) Diacritical marks (such as è, ñ, ç)

- What (if any) tests or verification is provided to ensure the 3000 word dictionary list referenced in the testing section is suitable for the task, or appropriate to the intended audience?
- Why is a Guideline related to [Common Words | Clear Words | Plain Language | Readability] mandating specific behaviors for modal popups?
("...disable the background text (use a modal display) and ensure that background text is covered or blurred so people aren't confused between foreground and background text, and that the keyboard focus remains in the modal until it is closed.") 

_[While noting that these are all best practices for the use of modals, where are these requirements normatively specified?]_
